event/event.go

```diff
@@ -9,6 +9,8 @@ type ImageUploaded struct {
 
 // ImagePublished provides an avro structure for an image published event
 type ImagePublished struct {
-	SrcPath string `avro:"src_path"`
-	DstPath string `avro:"dst_path"`
+	SrcPath      string `avro:"src_path"`
+	DstPath      string `avro:"dst_path"`
+	ImageID      string `avro:"image_id"`
+	ImageVariant string `avro:"image_variant"`
```

Comment:
Are there any unit tests for the Avro marshalling that can be updated to take into account the new fields that have been added to this model?

Reply:
Yes, the Image Published handler tests that the Id and Variant are passed through from POST to event. There are however, tests in `event/producer_test.go` that didn't have the new fields added for completeness so I've added them in now.